src/ui/server/controllers/insurance/account/sign-in/enter-code/index.ts

```diff
@@ -0,0 +1,56 @@
+import { PAGES } from '../../../../../content-strings';
+import { FIELD_IDS, ROUTES, TEMPLATES } from '../../../../../constants';
+import { ACCOUNT_FIELDS as FIELDS } from '../../../../../content-strings/fields/insurance/account';
+import insuranceCorePageVariables from '../../../../../helpers/page-variables/core/insurance';
+import { Request, Response } from '../../../../../../types';
+
+const {
+  ACCOUNT: { SECURITY_CODE },
+} = FIELD_IDS.INSURANCE;
+
+const {
+  INSURANCE: { DASHBOARD },
+} = ROUTES;
+
+/**
+ * PAGE_VARIABLES
+ * Page fields
+ * @returns {Object} Page variables
+ */
+export const PAGE_VARIABLES = {
+  FIELDS: {
```

Comment:
do we need to do this if there's only one field on the page?

Reply:
Good spot, I did have a go with single input variables function, had some troubles would have entailed a lot of changes due to the way the function gets field strings with the current strings structure (not all insurance fields are available at the root).

Will have another look, have another idea of just using core page variables or tweaking the field string imports

Reply:
Updated this so there is no longer `FIELDS` and just `FIELD`. Tried to use single input variables function, but it entails a lot of work. We will probably have same/similar requirement when it comes to declarations, can look at the function more then.